src/Diagnostics.RuntimeHost/Controllers/DiagnosticControllerBase.cs

```diff
@@ -500,6 +544,11 @@ protected async Task<IActionResult> GetInsights(TResource resource, string pesId
 
         #endregion API Response Methods
 
+        protected bool IsLocalizationApplicable(string language)
```

Comment:
I think you should move this method as well to TranslatorService (In case you need to repurpose this in some other components in future)